### DISCUSSION TITLE
OODA Loops For Radars

### DISCUSSION THREAD

**Discussion by HVAA** [PygmalionOfCyprus/cmo-db-requests#1625] [Q&A] [answered]
I was looking at some of the OODA loops for radars with a longish scan (ie more than 10 RPM) and I notice that many have loops of less than 2 scans.  As a former radar operator, this seems to be overly generous to say the least.  I was wondering what your take on this before I commit to finding sources you can use and asking to correct a whole host of radars.
So this is the physics of it.
Scan 1, target paints, circuitry places target in a buffer to determine whether the target is a true moving target or a piece of clutter.
Scan 2, target paints, the system then compares target from scan 1 to scan 2 to determine if the target is a valid one by using range comparison.
This is a really generous MTI circuit as some will hold returns in the loop for longer.
So at target unscreen, the radar has just painted that bit of sky a second prior.  Scan 1 10 seconds, Scan 2 10 seconds, even then with instantaneous processing, the OODA should be at least 20 seconds, not lower numbers that I am seeing, which is leading to near instant pick up of air targets, especially small, slower ones, which should cause these radars more issues than they seem to at present.

**Comment by PygmalionOfCyprus** [comment, marked as answer]
Working on it! The last of the OODA overhauls (for facilities and submarines) is coming in 496.

**Comment by PygmalionOfCyprus** [comment]
Since you have experience in this field, what would you say is a reasonable "average" detection time for radars, in general? I understand that scan time is a big determinant, but I'm looking to set the "default" CSGen values for radars; in our current implementation we have two different generations for "mechanical" and "electronic" sensors, so your thoughts on both would be appreciated.

**Comment by HVAA** [comment]
Its going to be about 3 times the RPM of the radar for older, mechanically scanned radars and perhaps even some of the newer ones that don't have the ability to 'squint' beams off the azimuth boresight but nonetheless have an elevation scan on them making them 3D radars (examples here might be RAT-31, Snow Drift or Top Plate) and seem like they should be electronic.
True electronically scanned radars will 'revisit' portions of the sky containing targets far more frequently so they will have a much reduced OODA with the true abiltiy to quickly determine 'pop ups'.  Examples here would be Sampson, Arabel etc.

So to answer your question, if you are working with just 2 models, then it does not seem unreasonable for you to apply 25 to 30 seconds for radars with older scanning mechanisms and perhaps 10 - 15 seconds for those with better methods of scanning the sky.  The bottom line is it takes a little while to detect, classify as a target and then declare as a track.  Once the autotrackers are on, then the way I see radars paint and update 'Bogey's' etc seems to be pretty much spot on (as in it matches a reasonable scan period for the radar concerned).

The only issue I would worry about is that if you have to use this method for any form of Fire Control Radar.  Ideally they would be cued (other radar, passive sensor or Mk1 Eye Ball), so the track declaration is made for them.
